packages/xdl/src/Analytics.ts

```diff
@@ -87,6 +90,9 @@ export class AnalyticsClient {
   }
 }
 
-const defaultClient = new AnalyticsClient();
+const AnalyticsUnified = new AnalyticsClient();
+AnalyticsUnified.setSegmentNodeKey('u4e9dmCiNpwIZTXuyZPOJE7KjCMowdx5');
```

Comment:
maybe lets keep setSegmentNodeKey and setVersionName calls at the same place

Reply:
thought you asked to pull them in, sry! i agree they should be the same place